README.md

```diff
@@ -9,20 +9,24 @@ speaks, see:
 Running locally
 ---------------
 
-For dev testing you probably want to run from a virtualenv::
+For dev testing, create and activate a virtual environment:
 
-    virtualenv shavar
+    virtualenv -p python3.7 shavar
     source shavar/bin/activate
 
 Install the necessary dependencies:
 
-    pip install -r requirements.txt
+    pip install -r requirements-test.txt
 
-Run unit tests to be sure it isn' horribly broken::
+Run code style check:
 
-    make test
+    flake8 --exclude ./shavar/lib,./shavar/bin,./build,./deactivate,./.local
```

Comment:
I get some additional errors:
```
./setup.py:17:9: E121 continuation line under-indented for hanging indent
./setup.py:21:9: E123 closing bracket does not match indentation of opening bracket's line
./mozsvc/util.py:11:22: E401 multiple imports on one line
./mozsvc/util.py:133:74: E226 missing whitespace around arithmetic operator
./mozsvc/http_helpers.py:6:22: E401 multiple imports on one line
./mozsvc/tests/test_user.py:33:1: F841 local variable 'err' is assigned to but never used
./mozsvc/tests/test_http_helpers.py:7:22: E401 multiple imports on one line
./mozsvc/tests/test_config.py:194:27: E127 continuation line over-indented for visual indent
./mozsvc/storage/mcclient.py:78:17: F841 local variable 'err' is assigned to but never used
```

It may be easy to fix these, or change the command to be closer to the ``Makefile``:

```
flake8 --exclude ./shavar/lib,./shavar/bin,./build,./deactivate,./.local shavar
```

This could all be moved to `setup.cfg`, so a plain `flake8` would work:

```
[flake8]
ignore = W503,E241
include =
    shavar
exclude = 
    ./shavar/lib,
    ./shavar/bin,
    ./build,
    ./deactivate,
    ./.local
```